glpi-project/glpi
# Clean code related to sanitization

| Q             | A
| ------------- | ---
| Bug fix?      | probably
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

- Deprecate some methods
- Remove useless sanitize/unsanitize operations on add/update
- Remove code related to global sanitization handling that is no longer needed
- Replace sanitization methods used for display by `htmlspecialchars()` / `json_encode()`
